client/packages/invoices/src/InboundShipment/DetailView/DetailView.tsx

```diff
@@ -55,17 +55,30 @@ export const DetailView: FC = () => {
     [onOpen]
   );
 
-  const onReturn = async (selectedStockLineIds: string[]) => {
+  const onReturn = async (selectedLines: InboundLineFragment[]) => {
     if (!data || !canReturnInboundLines(data)) {
       const cantReturnSnack = info(t('messages.cant-return-shipment'));
       cantReturnSnack();
-    } else if (!selectedStockLineIds.length) {
+      return;
+    }
+    if (!selectedLines.length) {
       const selectLinesSnack = info(t('messages.select-rows-to-return'));
       selectLinesSnack();
-    } else {
-      onOpenReturns(selectedStockLineIds);
-      setReturnMode(ModalMode.Create);
+      return;
+    }
+    if (selectedLines.some(line => !line.stockLine)) {
+      console.error('No stock line associated with the selected line(s)');
+      const selectLinesSnack = error(t('error.something-wrong'));
```

Comment:
Keen for thoughts here... this isn't an error the user could do anything about, and it kind of confusing to explain... so just went with generic error, and slightly more detail to the console. Should I provide more/less anywhere?

Reply:
I think this would be an edge case with inbound shipments that have been created in og store.

My preference is to show to the user, we have no access to console in production, in that case I was wondering what value would console.error bring ? 

I think it's ok to show untranslated error concatenated with error.something-wrong

Reply:
Ah ok perfect, will do :)